packages/terra-graphs-docs/src/example-datasets/graphConfigObjects/Gnatt/ganttDefault.js

```diff
@@ -0,0 +1,168 @@
+import Carbon from '@cerner/carbon-graphs/lib/js/carbon';
+import {
+  loadDatelinePopup,
+} from '@cerner/terra-graphs-docs/lib/example-datasets/popup';
+
+const daysToMilliseconds = (d) => 24 * 60 * 60 * 1000 * d;
+const lineDefault = (id) => ({
```

Comment:
update function name. 
Nit: I would name it to something like `getGanttDefaultConfig`

Reply:
Updated [here](https://github.com/cerner/terra-graphs/pull/133/commits/f67558128110f7b4e7d30699ab322b6245e34f54#diff-26d100f2bc9ed50b5b1d850422c6b831fee7e3b649854f848d613e9824f5f057R7).